servers/src/main/java/tachyon/master/journal/JsonJournalFormatter.java

```diff
@@ -356,6 +373,48 @@ public JournalEntry getNextEntry() throws IOException {
                 entry.getLong("id"),
                 entry.getByteBuffer("metadata"));
           }
+
+          // Lineage
+          case ASYNC_COMPLETE_FILE: {
+            return new AsyncCompleteFileEntry(
+                entry.getLong("fileId"));
+          }
+          case PERSIST_FILE: {
+            return new PersistFilesEntry(
+                entry.get("fileIds", new TypeReference<List<Long>>() {
+                }));
+          }
+          case LINEAGE: {
+            List<TachyonFile> inputFiles = Lists.newArrayList();
+            for (long fileId : entry.get("inputFiles", new TypeReference<List<Long>>() {})) {
+              inputFiles.add(new TachyonFile(fileId));
```

Comment:
Why do we wrap the `fileId` in a `TachyonFile`? I thought `TachyonFile` is just a client side concept?